introduction-to-datasets.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+URIs also define a name and namespace for your dataset. Kite uses these values when the underlying system has the same concept (for example, Hive). You can also retrieve the values using the `Datasets.list` method.
+
+Often, the name and namespace provide a location that could be an HDFS path.
+
+To ensure compatibility with underlying systems, names and namespaces in URIs must be mode of alphanumeric characters or underscore (_) and cannot start with a number.
+
+You can create datasets in Hive, HDFS, HBase, or as local files. There are several URI patterns available and additional options.See [Dataset and View URIs]({{site.baseurl}}/URIs.html).
+
+## Schemas
+
+A schema defines the field names and datatypes for a dataset. Kite relies on an Apache Avro schema definition for each dataset. For example, this is the schema definition for a table using four columns from the `movies` dataset.[<sup>1</sup>](#notes)
```

Comment:
We chose to standardize on Avro for a few reasons:
* The format is widely used, supported across components (as Tom noted), and mature
* Avro supports all 3 data models we're interested in exposing (generic, reflect, specific)
* Avro Schemas work with both Avro and Parquet, the main data formats we're interested in
* Avro Schemas are expressive enough and we don't hear too many complaints about lack of some structures or concepts

That's obviously way too much information to put here, but I'd like to see some justification rather than just stating that it is the case.